### PR TITLE
Merged SLE-12 branch

### PR DESCRIPTION
**Pull request by mchf**

### CODE REVIEW

**Comment by mvidner** [comment]
So how does this one relate to #365 which is in an "after_release" branch?

File: src/include/network/lan/complex.rb [hunk 1]
```diff
@@ -443,26 +443,24 @@ def handleOverview(_key, event)
...
-            "STARTMODE"
-            ) == "nfsroot"
-            if !Popup.YesNoHeadline(
-              Label.WarningMsg,
-              _("Device you select has STARTMODE=nfsroot. Really delete?")
-              )
-              # y2r: cannot break from middle of switch
-              # but in this function return will do
-              return nil
+            return :edit
```
Comment (jreidinger) [inline src/include/network/lan/complex.rb]: indentation here looks wrong, also in whole file

**Comment by mchf** [comment]
Well, I'm not author of "after release" branch, so this one duplicates the content (I'll ask @imobach to check it). As there is update project already open in IBS I don't see a reason for "after release branch"

File: src/include/network/lan/complex.rb (hunk 1 above)
Comment (mchf) [inline src/include/network/lan/complex.rb]: yes it is. Indentation is fixed in SLE-12 branch but not in SP1. So it looks weird. I expect rubocop will complain, so I'll fix it with separate commit.

**Comment by imobach** [comment]
It's fine with me. I've created `sle-12-sp1-after_release` branches in some repositories following the agreement on ML (`SLE12-SP1: Post-RC2 Behavior`). We should take this into account when merging that branches to avoid duplications in changes file.

**Comment by mvidner** [comment]
LGTM

**Comment by mchf** [comment]
thanks for reviews

**Pull request merged**